js/modules/k6/metrics/metrics_test.go

```diff
@@ -43,13 +43,19 @@ func TestMetrics(t *testing.T) {
 		"Rate":    stats.Rate,
 	}
 	values := map[string]struct {
-		JS    string
-		Float float64
+		JS      string
+		Float   float64
+		isError bool
 	}{
-		"Float": {`2.5`, 2.5},
-		"Int":   {`5`, 5.0},
-		"True":  {`true`, 1.0},
-		"False": {`false`, 0.0},
+		"Float":     {JS: `2.5`, Float: 2.5},
+		"Int":       {JS: `5`, Float: 5.0},
+		"True":      {JS: `true`, Float: 1.0},
+		"False":     {JS: `false`, Float: 0.0},
+		"null":      {JS: `null`, isError: true},
+		"undefined": {JS: `undefined`, isError: true},
+		"NaN":       {JS: `NaN`, isError: true},
+		"string":    {JS: `"string"`, isError: true},
+		"string 5":  {JS: `"5.3"`, Float: 5.3},
```

Comment:
This also fails with adding another metric object, but it looks like it will require some rewriting so I have skipped it for now